libs/agno/agno/agent/agent.py

```diff
@@ -910,6 +914,8 @@ def run(
 
                     # Do a final check confirming the content is in the response_model format
                     if isinstance(run_response.content, self.response_model):
+                        # Restore original show_tool_calls value
+                        self.show_tool_calls = original_show_tool_calls
```

Comment:
We probably don't need to do this? We don't need to set it to false anymore with structured output